lib/rubocop/cop/layout/space_inside_parens.rb

```diff
@@ -33,6 +33,27 @@ module Layout
       #   g = ( a + 3 )
       #   y()
       #
+      # @example EnforcedStyle: compact
+      #   # The `compact` style enforces that parentheses have a space at the
+      #   # beginning with the exception that successive right parentheses are allowed.
```

Comment:
I think you should remove "right" here. It reads better to me without it.

Reply:
On it.

Reply:
Done.